waspc/data/Generator/templates/react-app/src/auth/useAuth.ts

```diff
@@ -23,5 +27,8 @@ export async function getMe(): Promise<User | null> {
   }
 }
 
-getMe.queryCacheKey = ['auth/me']
-getMe.route = { method: HttpMethod.Get, path: '/auth/me' }
+addMetadataToQuery(getMe, {
```

Comment:
Placing this on the module's top level does work, but feels a bit janky. Maybe we should put the entire initialization process in a function that returns `getMe`, and then export that.